src/Exports.jl

```diff
@@ -153,7 +153,9 @@ export divisible
 export divisor_lenstra
 export divisor_sigma
 export divisors
-export eigenvalues
+export eigenspace
+export eigenspaces
+export eigenvalues_with_multiplicities
 export eigvals
 export eigvals_simple
```

Comment:
If these are the names from LinearAlgebra, they shouldn't be exported. Instead, export the new names. (Again following how AA does it here)